src/main/java/equipments/AbstractEquipment.java

```diff
@@ -0,0 +1,9 @@
+package equipments;
+
+public abstract class AbstractEquipment implements IEquipment {
```

Comment:
is anything ever going to implement `IEquipment` without also inheriting from  `AbstractEquipment`? if not, use abstract methods instead